robocoder99/sonic-buildimage
# [device][mellanox] Fix supported breakout modes on MSN4410

#### Why I did it
The default breakout mode according to `hwsku.json` for the MSN4410 is `1x400G` and this is not a supported breakout mode according to its `platform.json`

This causes a conflict on boot of this platform and no containers on the switch will init successfully.  

#### How I did it
Referenced the platform specification files and updated `platform.json`

#### How to verify it
1. Install master version of SONiC on MSN4410
2. Boot switch and verify swss is successfully running using `docker ps`

#### Description for the changelog
[device][mellanox] Fix supported breakout modes on MSN4410

#### A picture of a cute animal (not mandatory but encouraged)
![Chicks](https://user-images.githubusercontent.com/5898707/120402609-931d5c00-c310-11eb-8fe9-bfb924d827f5.png)
